pkg/cloudevents/transport/http/transport.go

```diff
@@ -300,7 +302,7 @@ func (t *Transport) StartReceiver(ctx context.Context) error {
 	defer t.reMu.Unlock()
 
 	if t.LongPollReq != nil {
-		go func() { _ = t.longPollStart(ctx) }()
```

Comment:
Why this change?

Reply:
The pure long pull client do not need listen any ports, so do not need the another go routine

Reply:
The pure long pull client do not need listen any ports, so do not need the another go routine

Reply:
The pure long pull client do not need listen any ports, so do not need the another go routine

Reply:
The pure long pull client do not need listen any ports, so do not need the another go routine

Reply:
The pure long pull client do not need listen any ports, so do not need the another go routine